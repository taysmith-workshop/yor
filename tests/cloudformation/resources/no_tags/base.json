{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "yor_trace",
            "Value": "fb67b27e-ade5-4c68-9bdf-81242d8346a1"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}