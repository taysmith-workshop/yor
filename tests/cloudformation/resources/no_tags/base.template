AWSTemplateFormatVersion: 2010-09-09
Resources:
  S3Bucket:
    Type: AWS::S3::Bucket
    Properties:
      AccessControl: PublicRead
      WebsiteConfiguration:
        IndexDocument: index.html
        ErrorDocument: error.html
      Tags:
        - Key: yor_trace
          Value: c5fd4e79-34e0-4463-8367-9ef9149488d5
    DeletionPolicy: Retain