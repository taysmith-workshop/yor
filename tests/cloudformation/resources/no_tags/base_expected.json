{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          },
          {
            "Key": "yor_trace",
            "Value": "76e1dfd1-9b59-44b1-ba11-bc7cdc24cde9"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}